index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+        <header class="main-header"> 
+            <a class="logo" href="#"><img src="images/My Logo (1).png" alt="home page logo"></a>
+            <nav class="nav-container">
+            <a class="menu" href="#" ><img src="icon/menu.png"alt="menu"></a>    
+        </nav>
+    </header>
+
+
+
+        <section class="part1">
+        <div>
```

Comment:
- [Optional] You did a great job addressing the requested changes @mumbereh. I have a suggestion to make though. As you can see there is a color difference between the navigation and the heading section of your page. this might seem minor but as a front-end engineers, we are expected to deliver something identical to the original design. 

![image](https://user-images.githubusercontent.com/69077061/227138126-125efb28-3d4c-4a03-88a2-9416fcbeb11d.png)